JsonLD.php

```diff
@@ -38,6 +38,8 @@ class JsonLD
     /** Identifier for the merged graph */
     const MERGED_GRAPH = '@merged';
 
+    private static $remoteDocumentLoader;
```

Comment:
Could you please rename this to `$documentLoader` (documents don't necessarily have to be remote)